common/include/pcl/pcl_macros.h

```diff
@@ -100,7 +102,7 @@ namespace pcl
   using int_fast16_t [[deprecated("use std::int_fast16_t instead of pcl::int_fast16_t")]] = std::int_fast16_t;
 }
 
-#if defined _WIN32 && defined _MSC_VER
+#if defined _WIN32 && (defined _MSC_VER || defined __MINGW32__)
```

Comment:
Would it be better to simply use `defined _WIN32` and no compiler check?

Reply:
I considered it, but chose to take the safe route just in case. But now I notice that the `#if` actually covers more than just the math defines below it, so I'll have to have another look at it. Especially considering your comments above..! I'll do some more testing today.